### ISSUE TITLE
Typo in Makefile

### ISSUE THREAD

**Issue by dertester** [farconada/RF24Network#1]
Hi,
i try to implement my Pi into an existing Network of Arduino Nodes with maniacbugs Rf24 and RF24Network Lib.

I cloned this , make all and make install.
make: **\* No rule to make target `install-r24', needed by`install'.  Stop.

I guess there is a typo in line 76 of the makefile. I changed that and no error 3xInstalled.

===Personal question===
Could you pls. tell me if you got this running?
I just started and dont know how to use the examples on the pi coming from the arduino ide.

Do i have to compile them first?

I really would appreciate some help. Maybe you documented your work somethere or you can point me to some good sources.

**Comment by farconada** [comment]
Yes, I got it running. But I dont remember a lot of about this project, I did it a lot of time ago
